internal/cli/atlas/logs/logs.go

```diff
@@ -16,10 +16,12 @@ package logs
 
 import (
 	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/decryption"
 	"github.com/spf13/cobra"
 )
 
-func Builder() *cobra.Command {
+// MongoCLIBuilder is used in "mongocli atlas" to avoid adding new command to deprecated cli.
```

Comment:
I'm confused by the comments and the usage as they seem to contradict each other

Reply:
😅 I'll rewrite it basically to avoid adding it to mongocli altas but adding it only to atlas cli